packages/mermaid/src/docs/syntax/gitgraph.md

```diff
@@ -533,9 +534,11 @@ Usage example:
        commit
 ```
 
-Sometimes we may want to change the orientation. Currently, Mermaid supports two orientations: **Left to Right**(default) and **Top to Bottom**.
+Sometimes, we may want to change the orientation of the graph.
 
-In order to change the orientation from top to bottom i.e. branches lined horizontally, you need to add `TB` along with `gitGraph`.
+Mermaid also supports a **top to bottom** orientation. In this mode, the commits run from top to bottom of the graph and branches are arranged side-by-side.
```

Comment:
And here 
```md
In `TB` orientation commits run from top to bottom of the graph and branches are arranged side-by-side.
```
and the example.

Reply:
I've used headers in my latest commit instead to make this clearer.